bigframes/ml/llm.py

```diff
@@ -154,8 +154,11 @@ def _create_bqml_model(self):
             )
 
         if self.model_name not in _TEXT_GENERATOR_ENDPOINTS:
-            raise ValueError(
-                f"Model name {self.model_name} is not supported. We only support {', '.join(_TEXT_GENERATOR_ENDPOINTS)}."
+            warnings.warn(
+                f"Model name '{self.model_name}' is not supported. "
+                f"We currently support the following models: {', '.join(_TEXT_GENERATOR_ENDPOINTS)}. "
```

Comment:
Can we say "We currently support" -> "We are currently aware of"

Reply:
Updated. That sounds better, thanks.